chef.py

```diff
@@ -24,30 +25,70 @@
 sess.mount('http://', forever_adapter)
 sess.mount('https://', forever_adapter)
 
-ID_BLACKLIST = ["html", "by-device", "new", "quantum", "general"]
+ID_BLACKLIST = ["html", "by-device", "new", "quantum", "general", "by-level"]
+ARABIC_NAME_CATEGORY = {
+    "Physics": "الفيزياء",
+    "Biology": "الأحياء",
+    "Chemistry": "الكيمياء",
+    "Motion": "الحركة",
+    "Sound and Waves": "الصوت والأمواج",
+    "Work Energy and Power": "الشغل والطاقة",
+    "Earth Science": "علوم الأرض",
+    "Math": "الرياضيات",
+    "Heat and Thermodynamics": "الحرارة والديناميكا الحرارية",
+    "Quantum Phenomena": "فيزياء الكم",
+    "Light and Radiation": "الضوء والإشعاعات",
+    "Electricity Magnets and Circuits": "المغناطيس الكهربائي والدّارة الكهربائية",
+    "pH Scale": "مقياس معامل الحموضة pH",
+    "مقياس pH  : إبتدائي": "مقياس معامل الحموضة pH  : إبتدائي",
```

Comment:
Is one of the English PhET simulations using an Arabic title?

Reply:
Sorry do you mean in the English channel or in the Arabic channel? This variable here is to replace the original Arabic content name (English) in the PhET website to Arabic, since having English content names and Arabic content names in the Arabic channel makes the channel not consistent.

Reply:
Sorry, missed the notification on this reply! I meant the English channel. To clarify, I noticed that line 43 in the dict above has both an Arabic key and value, while the other items have English keys and Arabic values (as expected). What's also a bit surprising is that the Arabic key and value are different. 

I was expecting all the keys to be in English, and I was wondering if this meant that the English version of the PhET channel also had an Arabic title for this item.

Reply:
thank you for clarification! @kollivier Line 43 contains both Arabic key and value because Hiba pointed out that there are some misspellings in the translations shown in the PheT original Arabic website. This dict is only useful for the Arabic channel but not used in English PheT channels.

I notice that I put the space in line 43 in the wrong place. Instead of `A: B`, I put `A :B`. I will fix that with a new commit.

Reply:
actually I think the space is correct in line 43. It's displayed in the right way in my Sublime Text, so it might just because of the Arabic format..

Reply:
We discussed this via Slack, and now I know this item is used to correct a typo. :) Just noting it in the comment thread so there is a conclusion in case we ever come back and read this! Anyway, it might be good in the future to put a note in the code for special cases like this, but so long as it works as intended this is fine, so approving and merging!

Reply:
ah yes! thank you for the suggestion! I should have put comments there to make it more clear.